pydocstyle.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this file anymore?

Reply:
Will be deleted as part of file structure fix task.